src/components/Contentful/HubSpotForm.js

```diff
@@ -24,6 +24,11 @@ export default class ContentfulHubSpotForm extends React.Component {
   }
 
   render() {
-    return <div className="contentful-hub-spot-form" id={this.props.id} />
+    return (
+      <div
+        className={'contentful-hub-spot-form ' + this.props.buttonColour}
```

Comment:
Could you put a test around this that checks that the buttonColour prop is passed through correctly?
I understand we're missing a fair few tests in the project, but should boy-scout and add tests when we touch something.